modules/ipi-install-preparing-the-provisioner-node-for-openshift-install.adoc

```diff
@@ -134,7 +134,7 @@ $ sudo nohup bash -c "
     nmcli con delete \"$PROV_CONN\"
     nmcli connection add ifname provisioning type bridge con-name provisioning
     nmcli con add type bridge-slave ifname \"$PROV_CONN\" master provisioning
-    nmcli connection modify provisioning ipv6.addresses fd00:1101::1/64 ipv6.method manual
+    nmcli connection modify provisioning ipv4.addresses 172.22.0.1/24 ipv4.method manual ipv6.addresses fd00:1101::1/64 ipv6.method manual
```

Comment:
```suggestion
    nmcli connection modify provisioning ipv4.addresses 172.22.0.1/24 ipv4.method \
  manual ipv6.addresses fd00:1101::1/64 ipv6.method manual
```
You might want to insert a backslash and line break as recommended by the "[Formatting](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#formatting)" section in the Supplemental Style Guide:
"For long lines of code that you want to break up among multiple lines, use a backslash to show the line break. For example:

$ oc get endpoints --all-namespaces --template \
    '{{ range .items }}{{ .metadata.namespace }}:{{ .metadata.name }} \
    {{ range .subsets }}{{ range .addresses }}{{ .ip }} \
    {{ end }}{{ end }}{{ "\n" }}{{ end }}' | awk '/ 172\.30\./ { print $1 }'

Reply:
Done.